test/unit/resources/ssh_conf_test.rb

```diff
@@ -14,6 +14,12 @@
       _(resource.Tunnel).must_equal nil
       _(resource.SendEnv).must_equal 'LANG LC_*'
       _(resource.HashKnownHosts).must_equal 'yes'
+      puts resource
+    end
+
+    it 'is case insensitive' do
+      resource = load_resource('ssh_config')
+      _(resource.gssapiauthentication).must_equal 'no'
```

Comment:
You might consider adding an assertion here testing other capitalizations.  This test would fail to catch a regression where we only accepted lowercase specifications.